recipe/meta.yaml

```diff
@@ -73,39 +76,29 @@ requirements:
     - libboost-devel
     - pyyaml
     - ply
+    - libpisp   # [variant == "rpi_fork"]
   run:
     - python
 test:
   commands:
+    # command only available in the upstream fork
+    - cam -l  # [variant == "upstream"]
```

Comment:
this doesn't seem true. it seems that the libcamera-rpi also has it
https://github.com/raspberrypi/libcamera/tree/main/src/apps

can we find out why it is disabled?

Reply:
you've disabled a bunch of meson options in your build script, can you please work to enable them back in the rpi variant?

Reply:
looks like it's a configurable option: https://github.com/raspberrypi/libcamera/blob/main/meson_options.txt

Reply:
yes... you disabled it when you first merged the recipe. I enabled it back.